source/includes/_egress.md

```diff
@@ -86,3 +86,117 @@ Field       | Type    | Description
 ----------- | ------- | -----------
 invoice_id  | String  | Internal **ON/RAMP**'s Invoice Identifier.
 invoice_url | String  | Url where to redirect user.
+
+
+#### Manual flow
```

Comment:
Maybe calling "Manual" could be a bit misleading, as it does not imply to be do anything manual; and actually both flows can be done automatic or manual (though for the first one it would require to set the state to the undocumented postponed).

What about something like "app-redirect flow" and "user email flow"?

Reply:
Sure, that could work too and be less misleading. I'll update it.

Reply:
I updated it!